### PR TITLE
Update Safari Technology Preview to 115

### PR DESCRIPTION
**Pull request by autofoolip**
Version as reported by safaridriver --version:
Included with Safari Technology Preview (Release 115, 15611.1.3.5)

Source: https://developer.apple.com/safari/download/
Build: https://dev.azure.com/foolip/safari-technology-preview-updater/_build/results?buildId=2263&view=logs

### CODE REVIEW

**Review by gsnedders — APPROVED** [review]
macOS Safari infrastructure/ tests got expected results, merging on that basis

**Comment by foolip** [comment]
Results are in: https://wpt.fyi/results/?sha=8b1df3d520335fb42ef4bfd50b9121ac41995f8e

**Comment by foolip** [comment]
https://wpt.fyi/results/?diff&filter=ADC&run_id=716760001&run_id=732280003 looks pretty great, in particular:
- [flex gap](https://wpt.fyi/results/css/css-flexbox?diff&filter=ADC&run_id=716760001&run_id=732280003) (thanks @svillar!)
- [`<input type=date>`](https://wpt.fyi/results/html/semantics/forms/constraints/form-validation-checkValidity.html?diff&filter=ADC&run_id=716760001&run_id=732280003)!? [I've been wondering about that...](https://twitter.com/foolip/status/1273293516739940354)
- [Unprefixed Web Audio!](https://wpt.fyi/results/webaudio?diff&filter=ADC&q=is%3Adifferent&run_id=716760001&run_id=732280003)

And lots of other stuff that I'm not as familiar with.

Screenshot proof of `<input type=date>`:
image

**Comment by foolip** [comment]
Hmm, only one of those (unprefixed Web Audio) is mentioned in https://developer.apple.com/safari/technology-preview/release-notes/ @gsnedders @apple-web-evangelist FYI.

**Comment by Schweinepriester** [comment]
@foolip `<input type=date>` was mentioned in TP 114, but not actually there, came with 115.
We did some testing starting in https://github.com/Fyrd/caniuse/pull/5646#issuecomment-706249532, resulting in https://github.com/Fyrd/caniuse/pull/5669 now.

TL;DR: UI for `type="datetime-local"` and `type="date"` is there, for `type="time"` kinda 🎉
